pkg/apis/openebs.io/lvm/v1alpha1/lvmsnapshot.go

```diff
@@ -49,6 +49,18 @@ type LVMSnapshotSpec struct {
 	// SnapSize specifies the space reserved for the snapshot
 	// +kubebuilder:validation:Required
 	SnapSize string `json:"snapSize,omitempty"`
+
+	// SourceVolumeCapacity represents the capacity of the source volume
```

Comment:
In general, use the word `origin` to refer to this volume.

```suggestion
	// SourceVolumeCapacity represents the capacity of the origin volume
```